src/webgpu/util/texture/texelData.spec.ts

```diff
@@ -61,19 +73,15 @@ function doTest(
 
   [[block]] struct Output {
     ${rep.componentOrder
-      .map((C, i) => {
-        return `[[offset(${i * 4})]] result${C} : ${shaderType};`;
-      })
+      .map((C, i) => `[[offset(${i * 4})]] result${C} : ${shaderType};`)
       .join('\n')}
   };
   [[set(0), binding(1)]] var<storage_buffer> output : Output;
 
   [[stage(compute)]]
   fn main() -> void {
       ${rep.componentOrder
-        .map(C => {
-          return `output.result${C} = textureLoad(tex, vec2<i32>(0, 0), 0).${C.toLowerCase()};`;
-        })
+        .map(C => `output.result${C} = textureLoad(tex, vec2<i32>(0, 0), 0).${C.toLowerCase()};`)
```

Comment:
uNit: could textureLoad just once, then this code generation would be a little easier to read.

Reply:
Okay made this load once but then write into individual scalar slots of the struct. I guess I could have made the struct vecN<type> but it doesn't seem like it makes much of a difference.